flowclient/flowclient/client.py

```diff
@@ -935,3 +935,31 @@ def flows(
         "to_location": to_location,
         "aggregation_unit": aggregation_unit,
     }
+
+
+def unique_subscriber_counts(
+    start_date: str, stop_date: str, aggregation_unit: str
```

Comment:
Sorry, I missed this earlier: this parameter should actually be named `end_date` rather than `stop_date`. (This was caught by the most recent integration test you added, which is why they are currently failing.)